src/main.ts

```diff
@@ -32,6 +33,11 @@ const run = async () => {
             core.setFailed('Please enter a valid script.');
             return;
         }
+        console.log(`Starting CredScan.`);
+        let credscancheck: string = inlineScript;

```

Comment:
what is the need for this initialization?

Reply:
I'll remove it